### PR TITLE
upcoming: [M3-7704] - Disable Cloning, Private IP, Backups for edge regions

### PR DESCRIPTION
**Pull request by hana-linode**
## Description 📝
Cloning, Private IP, Backups, Lish Console via SSH, and VPC will not be supported for edge regions so we want to disable them.

## Changes  🔄
- Disabled the Backups and Private IP checkbox in the Linode Create page when an edge region is selected and display a warning notice
- Disabled Private IP radio in the "Add an IP Address" drawer (Linode details page -> Network tab -> Add An IP Address) for Linodes in an edge region with a warning notice
- Disabled the "Enable Backups" button in the Backups tab of the Linode details page with a warning notice
- Disabled the "Clone" action in the Linode landing dropdown action menu and Linode entity detail action menu with a tooltip explanation
- Updated LISH Console via SSH text in Linode Entity Detail Header to N/A

## Preview 📷

| image | image |
| ------- | ------- |
| image | image |
| image |

### Prerequisites
(How to setup test environment)
- Turn on the MSW

### Verification steps
(How to verify changes)
- See the Changes section

```
yarn test AddonsPanel
yarn test BackupsPlaceholder
yarn test AddIPDrawer
yarn test LinodeActionMenu
```

## As an Author I have considered 🤔

*Check all that apply*

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [x] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [x] 🧪 Providing/Improving test coverage
- [x] 🔐 Removing all sensitive information from the code and PR description
- [x] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/Linodes/LinodesDetail/LinodeBackup/BackupsPlaceholder.tsx [hunk 1]
```diff
@@ -42,14 +49,21 @@ export const BackupsPlaceholder = React.memo((props: Props) => {
   return (
     <>
       {disabled && <LinodePermissionsError />}
+      {linodeIsInEdgeRegion && (
+        <Notice
+          text="Backups are currently not available for Edge regions"
+          variant="warning"
+        />
+      )}
```
Comment (bnussman-akamai) [inline packages/manager/src/features/Linodes/LinodesDetail/LinodeBackup/BackupsPlaceholder.tsx:57]: I think the following pattern would look a lot cleaner than the notice. Would this make sense here?

Screenshot 2024-03-04 at 7 00 03 PM

File: packages/manager/src/features/Linodes/LinodesLanding/LinodeActionMenu/LinodeActionMenu.test.tsx [hunk 2]
```diff
@@ -125,6 +131,30 @@ describe('LinodeActionMenu', () => {
...
+        rest.get('*/regions', (req, res, ctx) => {
+          return res(ctx.json(makeResourcePage(regions)));
+        })
+      );
+
+      const { getByLabelText, getByTestId } = renderWithTheme(
+        <LinodeActionMenu {...propsWithEdge} />,
+        {
+          flags: { gecko: true },
+          queryClient,
```
[the remark this answers is not in the document]
Comment (hana-linode) [reply packages/manager/src/features/Linodes/LinodesLanding/LinodeActionMenu/LinodeActionMenu.test.tsx:151]: Nope, good catch!

File: packages/manager/src/features/Linodes/LinodesDetail/LinodeBackup/BackupsPlaceholder.tsx (hunk 1 above)
Comment (hana-linode) [reply packages/manager/src/features/Linodes/LinodesDetail/LinodeBackup/BackupsPlaceholder.tsx:57]: Makes sense, confirmed with Kendall on the change 🧹

**Review by dwiley-akamai — APPROVED** [review]
Unit tests pass ✅

Linode Create page: Backups & Private IP checkboxes disabled & warning notice displayed when Edge region is selected ✅ 

Disabled Private IP radio in the "Add an IP Address" drawer (Linode details page -> Network tab -> Add An IP Address) for Linodes in an Edge region with a warning notice ✅ 

Disabled the "Enable Backups" button in the Backups tab of the Linode details page with a warning notice ✅ 

Disabled the "Clone" action in the Linode landing dropdown action menu and Linode entity detail action menu with a tooltip explanation ✅

**Comment by jaalah-akamai** [comment]
Instead of a notice for the backups section. Can we add tooltip text to the disabled button? I think that's a cleaner approach that we've been doing elsewhere

**Comment by hana-linode** [comment]
@jaalah-akamai Backups has been addressed in an earlier commit, just updated the preview screenshot to reflect the change. For the "Add an IP Address" drawer, the user can still edit/modify most things in the drawer (just not Private IPv4) so I think the warning notice makes sense there

**Review by jaalah-akamai — APPROVED** [review]

**Pull request merged**